content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+Bevy programs store and execute all of their game logic and data with a single `App` data structure.
+Let's make a trivial Hello World app.
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::new()
+  .add_system(hello.system())
+  .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+The process is straightforward: we first create a new [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html).
+Then, we add a simple system, which prints "Hello, Bevy!" when it is run.
+Finally once we're done configuring the app, we call [`App::run()`](https://docs.rs/bevy/latest/bevy/app/struct.App.html#method.run) to actually make our app *do things*.
+
+## What makes an App?
+
+So, what sort of data does are `App` really store?
+Turns out: it's fairly simple.
```

Comment:
Alternatively, or maybe more appropriately, "Bevy programs store and execute all of their game logic and data with a single `App` data structure." should go here?